nix/default.nix

```diff
@@ -1,53 +1,54 @@
+{ system ? builtins.currentSystem }:
 let
-  pkgs = import ./nixpkgs.nix {
+  pkgs = (import ./nixpkgs.nix {
```

Comment:
The additional braces `()` should not be necessary.

Reply:
Please forgive this as I try to keep syntax sync with crun/conmon/etc too (simpler for diff across repo).